doc/Doxyfile.in

```diff
@@ -1485,7 +1485,7 @@ EXT_LINKS_IN_WINDOW    = NO
 
 FORMULA_FONTSIZE       = 10
 
-# Use the FORMULA_TRANPARENT tag to determine whether or not the images
+# Use the FORMULA_TRANSPARENT tag to determine whether or not the images
```

Comment:
The config *is* named `FORMULA_TRANSPARENT` so this is purely a typo correction, no logic effect.